src/main/java/org/folio/rest/impl/FinanceHelper.java

```diff
@@ -232,4 +328,27 @@ public void checkForCustomTransactionError(Throwable fail) {
     }
   }
 
+  /**
+   * Calculates remaining amount for encumbrance
+   * [remaining amount] = (allocated * allowableEncumbered) - (allocated - (unavailable + available)) - (encumbered + awaitingPayment + expenditures)
```

Comment:
OK, as @pautau-s brought to my attention on the wiki, this is actually incorrect.  I apologize for the confusion.  Please use:  `[remaining amount we can encumber] = (allocated * allowableEncumbered) - (encumbered + awaitingPayment + expended)`